### PR TITLE
[AWS Fargate] Update documentation

### LINKED ISSUE #9118: [AWS Fargate] Restructuring docs in accordance with documentation guidelines
Before releasing AWS Fargate as GA, the documentation has to be restructured to follow the [documentation guidelines](https://github.com/elastic/integrations/blob/main/docs/documentation_guidelines.md).

### PR DESCRIPTION
**Pull request by lucian-ioan**
## Proposed commit message

Update documentation and change docs stack version to 8.12.0.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] 

## How to test this PR locally
Run the elastic stack using `elastic-package` and set up the AWS Fargate integration (under beta integrations).

## Related issues

- Closes #9118

### CODE REVIEW

File: packages/awsfargate/_dev/build/docs/README.md [hunk 1]
```diff
@@ -24,30 +30,20 @@ TaskDefinition:
...
 This section shows you how to run the Elastic Agent in a ECS cluster, start collecting Fargate on ECS metrics, and send them to an Elastic Stack.
 
-First, we'll see a simple example, setting up a task definition and a service on an existing ECS cluster using the AWS web console; this is the quickest path to have the integration up and running in your existing ECS cluster.
-
-Second, we'll see a complete setup from scratch of a cluster, a service, and a task using a CloudFormation template and the AWS CLI.
-
-Let's get started!
-
-### Using the AWS web console
+Follow these steps for a fast deployment on your existing ECS cluster.
```
Comment (shmsr) [inline packages/awsfargate/_dev/build/docs/README.md:44]:
```suggestion
To quickly deploy on your existing ECS cluster, follow these steps.
```

File: packages/awsfargate/_dev/build/docs/README.md [hunk 2]
```diff
@@ -24,30 +30,20 @@ TaskDefinition:
...
 The Elastic Agent collects metrics using the [Amazon ECS task metadata endpoint](https://docs.aws.amazon.com/AmazonECS/latest/userguide/task-metadata-endpoint-fargate.html).
 
 The Amazon ECS task metadata endpoint is an HTTP endpoint available to each container and enabled by default on [AWS Fargate platform version 1.4.0](https://aws.amazon.com/blogs/containers/aws-fargate-launches-platform-version-1-4/) and later. The Elastic Agent uses [Task metadata endpoint version 4](https://docs.aws.amazon.com/AmazonECS/latest/userguide/task-metadata-endpoint-v4-fargate.html).
 
-## Credentials
-
-No AWS credentials are required for this integration. The ECS task metadata endpoint is accessible inside the cluster only.
-
-## Getting Started
+## Getting Started using the AWS web console
```
Comment (shmsr) [inline packages/awsfargate/_dev/build/docs/README.md:40]:
```
## Getting Started using the AWS web console
```

Isn't "AWS Management Console" is better than "AWS web console"?

Example: https://docs.aws.amazon.com/awsconsolehelpdocs/latest/gsg/learn-whats-new.html#console-device

If you are making this change, please replace all other usages "AWS web console".

File: packages/awsfargate/_dev/build/docs/README.md [hunk 3]
```diff
@@ -1,8 +1,14 @@
 # AWS Fargate Integration
 
+## Overview
+
 The AWS Fargate integration helps to retrieve metadata, network metrics, and Docker stats about your containers and the tasks that are a part of an [Amazon Elastic Container Service (Amazon ECS)](https://aws.amazon.com/ecs/?pg=ln&sec=hiw) cluster.
```
Comment (shmsr) [inline packages/awsfargate/_dev/build/docs/README.md:5]:
```suggestion
The AWS Fargate integration helps to retrieve metadata, network metrics, and Docker stats about your containers and the tasks that are part of an [Amazon Elastic Container Service (Amazon ECS)](https://aws.amazon.com/ecs/?pg=ln&sec=hiw) cluster.
```

**Review by shmsr — CHANGES_REQUESTED** [review]

File: packages/awsfargate/_dev/build/docs/README.md (hunk 1 above)
Comment (shmsr) [reply packages/awsfargate/_dev/build/docs/README.md:44]: nitpick: The usage of "fast" seems inappropriate. That's why I suggested the change.

File: packages/awsfargate/_dev/build/docs/README.md (hunk 2 above)
Comment (lucian-ioan) [reply packages/awsfargate/_dev/build/docs/README.md]: Definitely better, thank you!

**Review by shmsr — APPROVED** [review]

File: packages/awsfargate/docs/README.md [hunk 4]
```diff
@@ -24,32 +30,22 @@ TaskDefinition:
...
-### Using the AWS web console
+To quickly deploy on your existing ECS cluster, follow these steps.
 
-#### Task Definition
+### Task Definition
 
-Open the AWS web console and visit the Amazon ECS page. Here you can select "Task Definitions" and then "Create new Task Definition" to start the wizard.
+Open the AWS Management Console and visit the Amazon ECS page. Here you can select "Task Definitions" and then "Create new Task Definition" to start the wizard.
 
 In the step 1 select "Fargate" from the list of available launch types.
```
Comment (muthu-mps) [inline packages/awsfargate/docs/README.md:50]: Can we update the `steps` in a similar way? The `Step 1` and `Step 2` are not aligned properly.

Comment (lucian-ioan) [reply packages/awsfargate/docs/README.md]: Makes sense, thank you!

bullet_point

**Review by muthu-mps — APPROVED** [review]
LGTM!

File: packages/awsfargate/_dev/build/docs/README.md [hunk 5]
```diff
@@ -64,41 +60,41 @@ In the step 2:
...
 
-#### Service
+### Service
 
 Select an existing ECS cluster and create a new service with launch type "FARGATE". Use the task definition we just created.
 
 As soon as the Elastic Agent is started, open the dashboard "\[AWS Fargate\] Fargate Overview" and you will see the metrics show up in few minutes.
 
-### Using the AWS CLI
+## Getting Started using the AWS CLI
```
Comment (alaudazzi) [inline packages/awsfargate/_dev/build/docs/README.md:77]:
```suggestion
## Getting started using the AWS CLI
```

**Review by alaudazzi — APPROVED** [review]
Left a very minor edit, otherwise LGTM.

**Pull request merged**